Source/RimConnection/Settings/Dialog_Rename_ModOptions.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Verse;
+
+namespace RimConnection.Settings
+{
+    public class Dialog_Save_ModOptions : Dialog_Rename
```

Comment:
Not sure where to put this comment, but the modal for inputting the filename needs more information like where it's going to save to, a button to open the folder would be great if possible, something telling the user that it'll save as an xml and don't put the file extension on it. Potentially a success message on save that says where it's saved?

I like the load button though that's great, showing the list of files

Reply:
I'd potentially rename the button to something like "save/load settings" rather than file, that's a bit ambiguous